src/main/java/techcourse/myblog/domain/comment/Comment.java

```diff
@@ -1,18 +1,19 @@
-package techcourse.myblog.comment;
+package techcourse.myblog.domain.comment;
 
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import techcourse.myblog.date.Date;
-import techcourse.myblog.exception.InvalidAuthorException;
-import techcourse.myblog.user.User;
+import techcourse.myblog.domain.BaseEntity;
+import techcourse.myblog.domain.InvalidAuthorException;
+import techcourse.myblog.domain.article.Article;
+import techcourse.myblog.domain.user.User;
 
 import javax.persistence.*;
 
 @Entity
 @Getter
 @NoArgsConstructor
```

Comment:
JPA가 필요로 하는 기본 생성자는 private으로 만들어야 다른 개발자의 오용을 방지할 수 있어요. 아래의 글은 가볍게 참고만 해주세요.
https://goodyhlee.wordpress.com/2016/06/20/177

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
```